tests/integration/test_replay_events.py

```diff
@@ -38,7 +36,11 @@ def generate_replay_sdk_event():
                 "trace_id": "4C79F60C11214EB38604F4AE0781BFB2",
                 "span_id": "FA90FDEAD5F74052",
                 "type": "trace",
-            }
+            },
+            "replay": {
+                "error_sample_rate": 0.125,
+                "session_sample_rate": 0.5,
+            },
```

Comment:
This can go into a separate PR, but we should add this new Context to the protocol. see for example 

https://github.com/getsentry/relay/blob/23c24a93dd9c26749cbef6fc49a52d70d25e91bd/relay-general/src/protocol/contexts/browser.rs#L6-L16